databrowser/src/layouts/AppHeader.vue

```diff
@@ -1,12 +1,15 @@
 <template>
   <ContentAlignmentX class="flex flex-row justify-between py-3 lg:py-5">
-    <router-link to="/">
-      <img
-        src="/logo-open-data-hub-black.png"
-        :alt="$t('header.logo')"
-        class="logo"
-      />
-    </router-link>
+    <div class="flex items-center space-x-16">
+      <router-link to="/">
+        <img
+          :alt="$t('header.logo')"
+          class="logo"
+          src="/logo-open-data-hub-black.png"
+        />
+      </router-link>
+      <MenuPopover />
+    </div>
```

Comment:
On mobile the menu button and the logo are switches around.

<img width="704" alt="Bildschirmfoto 2022-01-12 um 13 18 17" src="https://user-images.githubusercontent.com/3337260/149138971-6f611dfb-c10a-42fa-8421-1cabccbf1ad1.png">